angular/projects/common/src/app/utils/record-constants.ts

```diff
@@ -2,15 +2,15 @@ import { Utils } from './utils';
 import { Legislation } from '../models/master/common-models/legislation';
 
 export class EpicProjectIds {
-  public static readonly lngCanadaId = '588510cdaaecd9001b815f84';
+  public static readonly lngCanadaId = '588511d0aaecd9001b826192';
```

Comment:
Accidental change?

Reply:
See https://bcmines.atlassian.net/browse/NRPT-157.

Reply:
Correct me if I followed that thread wrong: so this ID replaces the old one entirely? 
In that case, there are other instances of the old ID in the codebase that probably need replacing.  We would need a migration then as well to update the old IDs?

Reply:
I believe that is coming soon

Reply:
sweet 👍